openmetrics/tests/test_openmetrics.py

```diff
@@ -93,82 +100,24 @@ def test_openmetrics_wildcard(aggregator):
     aggregator.assert_all_metrics_covered()
 
 
-def test_openmetrics_default_instance(aggregator):
```

Comment:
The default instances/namespace concept is only for manual use by subclasses, the Agent will never have that for this check

![Capture](https://user-images.githubusercontent.com/9677399/105644219-57561f80-5e62-11eb-9fda-c4a44a0b73ca.PNG)